### ISSUE TITLE
Conda installation issue with Python 3.6

### ISSUE THREAD

**Issue by Garyfallidis** [cvxgrp/cvxpy#387]
Hi @SteveDiamond and all. How can I install cvxpy with conda and python 3.6?
Here is the exact error that I am getting in a windows machine with latest standard anaconda installation with python 3.6
```
$ conda install -c cvxgrp cvxpy
Fetching package metadata .............
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - cvxpy -> cvxcanon >=0.0.22 -> python 2.7*
  - python 3.6*
```

Thanks in advance.

**Comment by SteveDiamond** [comment]
Anaconda installation doesn't really work with Python 3 on windows. Python 2.7 works fine and Python 3.5 might work. You can also use Python(x,y).

**Comment by Garyfallidis** [comment]
Do you have any plans to support 3.6 with anaconda? I wouldn't want our users (I am part of the dipy.org crew) to have to change installation systems. We were very excited to see that you are provided these installers for 3.5.

**Comment by SteveDiamond** [comment]
Ok I made it installable on Windows with Python 3.6. SCS won't work for SDPs but otherwise things should be fine.

**Comment by Garyfallidis** [comment]
Oh great! Thank you @SteveDiamond . @arokem and @skoudoro I am tagging you here for reference. It seems cvxpy will work fine now with conda and 3.6 in windows.

**Comment by lpbrown999** [comment]
This is still an issue, installing in a fresh conda environment on python 3.6.8, MacOS 10.14, following https://www.cvxpy.org/install/index.html results in a broken installation. I was able to fix the problem by upgrading to numpy 1.15.4 from the required 1.13.1

**Comment by SteveDiamond** [comment]
I did my best to up the required numpy and fix this issue. Hopefully should work now.
